### PR TITLE
Bump Application Connectivity images

### PR DESCRIPTION
**Pull request by akgalwas**
**Description**

Changes proposed in this pull request:

- Bump Application Connectivity images

### CODE REVIEW

**Review by majakurcius — DISMISSED** [review]

**Review by koala7659 — APPROVED** [review]

**Review by rafalpotempa — DISMISSED** [review]

File: resources/application-connector/values.yaml [hunk 1]
```diff
@@ -23,37 +23,37 @@ global:
...
   application_gateway_legacy_tests:
-    version: "PR-9394"
+    version: "4eeef0b7"
   application_registry:
-    version: "PR-9406"
+    version: "2407bbc2"
   application_registry_tests:
-    version: "PR-9356"
+    version: "91ae6ff1"
   application_broker:
```
Comment (koala7659) [inline resources/application-connector/values.yaml:51]: Can we use version 2f501fd2 here?

**Review by koala7659 — APPROVED** [review]

File: resources/application-connector/values.yaml (hunk 1 above)
Comment (akgalwas) [reply resources/application-connector/values.yaml:51]: Let's leave it as it is for now and bump it later.

**Review by majakurcius — APPROVED** [review]

**Review by Szymongib — APPROVED** [review]

**Pull request merged**